### PR TITLE
Fixed invalid uni for new users

### PR DESCRIPTION
**Pull request by 2dev9**
Fixed the issue where new users could not make requests to grab coffee. Issue solved by inserting new users into the UniCollection database.

The VerifyUni method was deleted so users who remove themselves from the Columbia directory can still use the service.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 settings.json
 *.swp
 .uploads/*
+upload/*
```
Comment (alanhdu) [inline .gitignore]: Why do we need to ignore both `upload` and `.uploads`? Seems redundant.

File: node_modules/bcrypt/.meteor-portable-1.json [hunk 2]
```diff
@@ -0,0 +1 @@
+false
```
Comment (alanhdu) [inline node_modules/bcrypt/.meteor-portable-1.json]: Probably not a file we need in Github 😉

File: server/server.js [hunk 3]
```diff
@@ -40,10 +40,11 @@ Meteor.methods({
 
     var receiverEmail = PeopleCollection.findOne({ owner: receiver }).username;
 
-    if (senderUni !== null) {
+    if (senderUni != null) {
```
Comment (alanhdu) [inline server/server.js]: Hm... could you document why you this fixes the problem? It seems pretty non-obvious.

File: server/server.js [hunk 4]
```diff
@@ -40,10 +40,11 @@ Meteor.methods({
...
       // Check UNI cache first
-      var uni_details = UniCollection.find({ uni: senderUni }).fetch();
 
+      var uni_details = UniCollection.find({ uni: senderUni }).fetch();
+      console.log(uni_details);
```
Comment (alanhdu) [inline server/server.js]: This seems like a stray debugging.

File: .gitignore (hunk 1 above)
Comment (2dev9) [reply .gitignore]: They two different folders that hold profile pictures, which shouldn't be uploaded to Github

File: server/server.js (hunk 3 above)
Comment (2dev9) [reply server/server.js]: It doesn't but teammates agreed it was a better choice style-wise.

File: server/server.js (hunk 4 above)
Comment (2dev9) [reply server/server.js]: The log statement is. Thanks!

File: server/server.js (hunk 3 above)
Comment (alanhdu) [reply server/server.js]: Lol, that'd be why it's non obvious then 😆.

**Pull request merged**